app/src/RelayEnvironment.ts

```diff
@@ -9,7 +9,10 @@ import {
 function getSanitizedPath(path: string): string {
   return path.endsWith("/") ? path.slice(0, -1) : path;
 }
-const graphQLPath = getSanitizedPath(window.Config.basename) + "/graphql";
+const graphQLPath =
+  `${window.location.protocol}//${window.location.host}${getSanitizedPath(window.Config.basename)}` +
+  "/graphql";
```

Comment:
this is needed because basic auth comes in as http://{user}:{pass}asdfas.com but this needs to be stripped from the graphQL path